### ISSUE TITLE
Syntax errors while running brew install

### ISSUE THREAD

**Issue by bagashe** [brewsci/homebrew-bio#1153]
Thank you for maintaining this repo.

Looks like https://github.com/brewsci/homebrew-bio/commit/917f31de2ccd766fb2495d5f2bbb40772fa7431c got the merging wrong. I'm seeing Ruby syntax errors when using `brew install brewsci/bio/seqan@2` on Github actions' macOS containers. 

E.g.

```
==> Tapping brewsci/bio
Cloning into '/usr/local/Homebrew/Library/Taps/brewsci/homebrew-bio'...
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/star-aligner.rb
star-aligner: /usr/local/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/star-aligner.rb:19: syntax error, unexpected <<, expecting end
<<<<<<< HEAD
^~
/usr/local/Homebrew/Library/Taps/brewsci/homebrew-bio/Formula/star-aligner.rb:24: syntax error, unexpected ===, expecting end
=======
^~~
```
Is there a work around I can use while the issue is resolved?

Thanks again.

**Comment by SeekingMeaning** [comment]
Sorry about that, it should be fixed in c162edc755bb13fb3cd74b0accba83bfa39eb7af. I'm fairly certain that `brew tap` used to checkout `master` but a recent change to `brew` made it so the default branch is checked out (e.g. `main` vs `master`)

**Comment by bagashe** [comment]
Confirmed that the revert fixed this issue. Thanks for being so responsive.
